input-output/pydicom_series.py

```diff
@@ -301,6 +301,20 @@ def _getPixelDataFromDataset(ds):
 
 # The public functions and classes
 
+def find_shape(dataset):
+    """Find the expected shape of `dataset.pixel_array` without reading the pixel data"""
+    shape = dataset.Rows, dataset.Columns
+    samples = dataset.SamplesPerPixel
+    if samples > 1:
+        conf = dataset.PlanarConfiguration
+        if conf == 0:
+            shape += (samples,)
+        elif conf == 1:
+            shape = (samples,) + shape
+        else:
+            raise ValueError(f"Invalid Planar Configuration: '{conf}'")
+    return shape
+
```

Comment:
Given the previous discussion, here there is no NumberOfFrames - perhaps not needed for collecting individual slices across files, but for robustness, perhaps could be at least be checked and a NotImplemented thrown, or dealt with in some other way?

Reply:
Ok, will do